templates/assopy/invoice.html

```diff
@@ -1,6 +1,214 @@
-{% extends 'assopy/base_invoice.html' %}
+{% if not is_real_invoice %}
+    Placeholder text
```

Comment:
Can you replace this with "VAT invoices will be generated as soon as we have been issued a UK VAT ID." ?

Reply:
Or perhaps just "VAT ID" - this may happen again in the future.